sw/airborne/peripherals/vl53l1x_api.c

```diff
@@ -240,9 +245,31 @@ VL53L1X_ERROR VL53L1X_SensorInit(VL53L1_DEV dev)
   status = VL53L1X_StopRanging(dev);
   status = VL53L1_WrByte(dev, VL53L1_VHV_CONFIG__TIMEOUT_MACROP_LOOP_BOUND, 0x09); /* two bounds VHV */
   status = VL53L1_WrByte(dev, 0x0B, 0); /* start VHV from the previous temperature */
+#if VL53L1X_AUTO_INCR_ADDR
+  status = VL53L1X_SetI2CAddress(dev, auto_incr_addr);
+  auto_incr_addr += 2; // auto increment by 2 (+1 on 7 bits address)
+#endif
+  return status;
+}
+
+VL53L1X_ERROR VL53L1X_BootDevice(VL53L1_DEV dev, uint16_t TimingBudgetInMs, uint16_t DistanceMode, uint32_t InterMeasurementInMs)
+{
+  VL53L1X_ERROR status = 0;
+  uint8_t state;
+  do {
+    status = VL53L1X_BootState(dev, &state);
+  } while (!state);
+  status = VL53L1X_SensorInit(dev);
+  /* Configure sensor */
+  status = VL53L1X_SetTimingBudgetInMs(dev, TimingBudgetInMs);
+  status = VL53L1X_SetDistanceMode(dev, DistanceMode);
+  status = VL53L1X_SetInterMeasurementInMs(dev, InterMeasurementInMs);
+  /* Start measurement */
+  status = VL53L1X_StartRanging(dev);
```

Comment:
Not sure this would do, it is not a bitmask apparently. Anyway, I copied the other functions from the original lib but I agree this is completely stupid.

Reply:
It is not a bitmask, but the result can be tested as a bool I think (0 = ok, anything else is error) so or-ing should still make sense. I'd also be fine with completely removing the status and making the function return void.

Reply:
I prefer no status than a stupid one, so I changed to void

Reply:
Sounds good